maven-model-builder/src/main/java/org/apache/maven/model/building/DefaultModelBuilder.java

```diff
@@ -814,7 +814,7 @@ private Model readRawModel( ModelBuildingRequest request, DefaultModelProblemCol
             }
             catch ( IOException e )
             {
-                problems.add( new ModelProblemCollectorRequest( Severity.FATAL, Version.V37 ).setException( e ) );
```

Comment:
Very good catch. Is `V37` still used somewhere else?

Reply:
It shouldn't. If it is, it should be replaced with `40`

Reply:
There is no when this is merged.